addons/website/static/src/js/website.snippets.gallery.js

```diff
@@ -244,7 +250,10 @@ options.registry.gallery = options.Class.extend({
     },
     images_rm   : function (type) {
         if(type !== "click") return;
-        this.replace($('<div class="alert alert-info css_editable_mode_display"/>').text(_t("Add Images from the 'Customize' menu")));
+        var $text = $('<span class="o_add_images fa fa-plus-circle" style="cursor: pointer;">').text(_(" Add Images "))
+        var $add_img = $('<div class="alert alert-info css_editable_mode_display text-center"></div>');
+        this.replace($add_img);
+        $text.appendTo(this.$('.css_editable_mode_display'));
```

Comment:
Here is a more efficient way to code this using jQuery:
```javascript
var $addImg = $('<div>', {
    class: 'alert alert-info css_editable_mode_display text-center',
});
var $text = $('<span>', {
    class: 'o_add_images fa fa-plus-circle',
    style: 'cursor: pointer;',
    text: _t("Add Images"),
});
this.replace($addImg.append($text));
```

Note that I did not checked it so you should test. Also, you are using `_` instead of `_t` to translate so I think this is broken. You are also putting text in a `.fa` element, which is wrong as the text will have a different font (you did that the right way in the xml snippet template).